<launch>
    <arg name="map_name" default="active_map"/>

    <arg name="map_file_path" value="$(find ptR1_navigation)/maps/$(arg map_name).yaml"/>

    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file_path)" />

    <include file="$(find ptR1_navigation)/launch/amclV2.launch" />
</launch>